--- origsrc/nautilus-sendto-2.28.2/configure.in	2009-11-17 12:20:41.000000000 -0600
+++ src/nautilus-sendto-2.28.2/configure.in	2010-02-01 03:16:22.983078900 -0600
@@ -83,7 +83,7 @@ AC_SUBST([NAUTILUS_EXTENSION_DIR],[${ac_
 
 dnl Get details about evolution
 
-PKG_CHECK_MODULES(EVOLUTION, evolution-plugin < 2.27.1,
+PKG_CHECK_MODULES(EVOLUTION, evolution-plugin < 2.27.1 libglade-2.0,
 		  enable_evo=yes, enable_evo=no)
 AM_CONDITIONAL(HAVE_EVOLUTION, test "x$enable_evo" = "xyes")
 
--- origsrc/nautilus-sendto-2.28.2/evolution/Makefile.am	2009-05-05 06:26:13.000000000 -0500
+++ src/nautilus-sendto-2.28.2/evolution/Makefile.am	2010-02-01 14:21:13.942711300 -0600
@@ -8,8 +8,8 @@ plugin_DATA = org-gnome-evolution-send-a
 plugin_LTLIBRARIES = liborg-gnome-evolution-send-attachments-to.la
 
 liborg_gnome_evolution_send_attachments_to_la_SOURCES = nautilus-sendto.c
-liborg_gnome_evolution_send_attachments_to_la_LDFLAGS = -module -avoid-version $(NO_UNDEFINED)
-liborg_gnome_evolution_send_attachments_to_la_LIBADD = $(EVOLUTION_LIBS)
+liborg_gnome_evolution_send_attachments_to_la_LDFLAGS = -module -avoid-version -no-undefined -L$(plugindir)/../components
+liborg_gnome_evolution_send_attachments_to_la_LIBADD = $(EVOLUTION_LIBS) -levolution-mail
 
 EXTRA_DIST = org-gnome-evolution-send-attachments-to.eplug.xml plugin.mk
 
--- origsrc/nautilus-sendto-2.28.2/src/plugins/evolution/Makefile.am	2009-11-17 12:19:21.000000000 -0600
+++ src/nautilus-sendto-2.28.2/src/plugins/evolution/Makefile.am	2010-02-01 15:54:31.127852000 -0600
@@ -27,8 +27,8 @@ regenerate-built-sources:
 plugin_LTLIBRARIES = libnstevolution.la
 
 libnstevolution_la_SOURCES = evolution.c $(CLA_FILES) $(MARSHALFILES)
-libnstevolution_la_LDFLAGS = -module -avoid-version
-libnstevolution_la_LIBADD = $(NST_EBOOK_LIBS)
+libnstevolution_la_LDFLAGS = -module -avoid-version -no-undefined
+libnstevolution_la_LIBADD = $(NAUTILUS_SENDTO_LIBS) $(NST_EBOOK_LIBS)
 
 EXTRA_DIST = $(CLA_FILES)
 CLEANFILES = econtactentry-marshal.c econtactentry-marshal.h
--- origsrc/nautilus-sendto-2.28.2/src/plugins/pidgin/Makefile.am	2009-11-17 12:19:30.000000000 -0600
+++ src/nautilus-sendto-2.28.2/src/plugins/pidgin/Makefile.am	2010-02-01 20:44:36.683392800 -0600
@@ -15,6 +15,6 @@ INCLUDES =                              
 plugin_LTLIBRARIES = libnstpidgin.la
 
 libnstpidgin_la_SOURCES = pidgin.c
-libnstpidgin_la_LDFLAGS = -module -avoid-version
-libnstpidgin_la_LIBADD = $(PIDGIN_LIBS)
+libnstpidgin_la_LDFLAGS = -module -avoid-version -no-undefined
+libnstpidgin_la_LIBADD = $(NAUTILUS_SENDTO_LIBS) $(PIDGIN_LIBS)
 
--- origsrc/nautilus-sendto-2.28.2/src/plugins/removable-devices/Makefile.am	2009-11-17 12:19:05.000000000 -0600
+++ src/nautilus-sendto-2.28.2/src/plugins/removable-devices/Makefile.am	2010-02-01 20:44:50.736196600 -0600
@@ -14,6 +14,6 @@ INCLUDES =                              
 plugin_LTLIBRARIES = libnstremovable_devices.la
 
 libnstremovable_devices_la_SOURCES = removable-devices.c
-libnstremovable_devices_la_LDFLAGS = -module -avoid-version
-libnstremovable_devices_la_LIBADD = $(GIO_LIBS) $(builddir)/../libnstcommon.la
+libnstremovable_devices_la_LDFLAGS = -module -avoid-version -no-undefined
+libnstremovable_devices_la_LIBADD = $(NAUTILUS_SENDTO_LIBS) $(GIO_LIBS) $(builddir)/../libnstcommon.la
 
